vnext/ReactUWP/Views/TextInputViewManager.cpp

```diff
@@ -393,5 +512,73 @@ YGMeasureFunc TextInputViewManager::GetYogaCustomMeasureFunc() const {
   return DefaultYogaSelfMeasureFunc;
 }
 
+void TextInputViewManager::TransferProperties(
+    XamlView oldView,
+    XamlView newView) {
+  __super::TransferProperties(oldView, newView);
+
+  bool copyToPasswordBox = oldView.try_as<winrt::TextBox>() != nullptr;
+
+  // sync common control properties
+  TransferProperty(oldView, newView, winrt::Control::FontSizeProperty());
+  TransferProperty(oldView, newView, winrt::Control::FontFamilyProperty());
+  TransferProperty(oldView, newView, winrt::Control::FontWeightProperty());
+  TransferProperty(oldView, newView, winrt::Control::FontStyleProperty());
+  TransferProperty(
+      oldView, newView, winrt::Control::CharacterSpacingProperty());
+  TransferProperty(
+      oldView, newView, winrt::Control::IsTextScaleFactorEnabledProperty());
+  TransferProperty(oldView, newView, winrt::Control::BackgroundProperty());
+  TransferProperty(oldView, newView, winrt::Control::BorderBrushProperty());
+  TransferProperty(oldView, newView, winrt::Control::BorderThicknessProperty());
+  TransferProperty(oldView, newView, winrt::Control::PaddingProperty());
+  TransferProperty(oldView, newView, winrt::Control::ForegroundProperty());
+  TransferProperty(oldView, newView, winrt::Control::TabIndexProperty());
+
+  // sync common properties between TextBox and PasswordBox
+  if (copyToPasswordBox) {
```

Comment:
>copyToPasswordBox [](start = 6, length = 17)

I think the logic has some problem. it's possible TextBox<->TextBox and PasswordBox<->PasswordBox. and likely you only handled TextBox<->PasswordBox

Reply:
On TextInputViewManager level, we only need to work with situation for TextBox <=> PasswordBox. Rest scenarios should just use the base class implementation of this function, I will add additional check.

---
In reply to: [321373313](https://github.com/microsoft/react-native-windows/pull/3075#discussion_r321373313) [](ancestors = 321373313)